core/src/kura.rs

```diff
@@ -991,4 +1039,26 @@ mod tests {
             .init()
             .unwrap();
     }
+
+    #[test]
+    #[should_panic]
+    fn write_then_read_wsv_from_disk_should_fail() {
+        let dir = tempfile::tempdir().unwrap();
+        let kura = Kura::blank_kura_for_testing_with_dir(&dir.path());
+        let mut wsv = WorldStateView::new(crate::World::default(), kura.clone());
+
+        std::thread::sleep(std::time::Duration::from_secs(1));
+        let loaded_data = kura.try_load_wsv().unwrap();
+    }
```

Comment:
```suggestion
    #[test]
    fn read_wsv_from_disk_without_write_should_fail() {
        let dir = tempfile::tempdir().unwrap();
        let kura = Kura::blank_kura_for_testing_with_dir(&dir.path());
        let mut wsv = WorldStateView::new(crate::World::default(), kura.clone());

        assert!(kura.try_load_wsv().is_none())
    }
```

Reply:
No preference here. 

The `#[should_panic]` approach conveys that this is a negative test case more clearly, but this is also shown in the name of the test.